redis/5.2/modules/ROOT/pages/redis-connector-studio.adoc

```diff
@@ -0,0 +1,233 @@
+= Use Studio to Configure Redis Connector 5.2 - Mule 4
+:page-aliases: connectors::redis/redis-connector-studio.adoc
+:page-aliases: connectors::redis/redis-connector-design-center.adoc
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure an input source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<configure-other-fields,Configure the other connector fields>>.
+
+When you run your app, you can view the app log as described in <<view-app-log,View the App Log>>.
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:redis-connector-reference.adoc[Redis Connector 5.2 Reference].
```

Comment:
Per the convo Valkyrie and I had about these xrefs, we think it might be better to simply name the other texts but not to make them live links, and then include the live links in See Also:
```suggestion
If you are new to configuring connectors in Studio, see Use Studio to Configure a Connector. If, after reading this topic, you need additional information about the connector fields, see Redis Connector 5.2 Reference.
```